config/initializers/scheduler.rb

```diff
@@ -0,0 +1,8 @@
+require 'rake'
+require 'rufus-scheduler'
+
+Rufus::Scheduler.singleton.every '7d' do
```

Comment:
How does this work for multiple server environments?  I've looked at this in the past and my understanding was that every 7 days, every server that is alive would then run this job.

Also we should probably add the guard for console/rake tasks https://github.com/jmettraux/rufus-scheduler#avoid-scheduling-when-running-the-ruby-on-rails-console

Reply:
Nice catch, added a tiny bit of duct tape to use Redis as a sort of locking mechanism.